diff --git a/cmake/Modules/Findrav1e.cmake b/cmake/Modules/Findrav1e.cmake
--- a/cmake/Modules/Findrav1e.cmake
+++ b/cmake/Modules/Findrav1e.cmake
@@ -32,7 +32,7 @@ if(NOT RAV1E_INCLUDE_DIR)
 endif()
 
 if(NOT RAV1E_LIBRARY)
-    find_library(RAV1E_LIBRARY NAMES rav1e PATHS ${_RAV1E_LIBDIR})
+    find_library(RAV1E_LIBRARY NAMES rav1e rav1e.dll PATHS ${_RAV1E_LIBDIR})
 endif()
 
 set(RAV1E_LIBRARIES ${RAV1E_LIBRARIES} ${RAV1E_LIBRARY} ${_RAV1E_LDFLAGS})
