diff --git a/src/read.c b/src/read.c
index c884018e..30898e31 100644
--- a/src/read.c
+++ b/src/read.c
@@ -4622,7 +4622,7 @@ static avifResult avifParse(avifDecoder * decoder)
 #endif
     avifBool needsTmap = AVIF_FALSE;
     avifBool tmapSeen = AVIF_FALSE;
-    avifFileType ftyp = {};
+    avifFileType ftyp = {0};

     for (;;) {
         // Read just enough to get the next box header (a max of 32 bytes)
